src/app/payment/paymentMethod/OpyPaymentMethod.tsx

```diff
@@ -0,0 +1,49 @@
+import { PaymentInitializeOptions } from '@bigcommerce/checkout-sdk';
+import React, { useCallback, FunctionComponent } from 'react';
+import { Omit } from 'utility-types';
+
+import { withLanguage, TranslatedString, WithLanguageProps } from '../../locale';
+import { LoadingOverlay } from '../../ui/loading';
+
+import getPaymentMethodName from './getPaymentMethodName';
+import HostedWidgetPaymentMethod , { HostedWidgetPaymentMethodProps } from './HostedWidgetPaymentMethod';
+
+export type OpyPaymentMethodProps = Omit<HostedWidgetPaymentMethodProps, 'containerId'>;
+
+const OpyPaymentMethod: FunctionComponent<OpyPaymentMethodProps & WithLanguageProps> = ({
+    initializePayment,
+    method,
+    language,
+    isInitializing = false,
+    ...rest
+}) => {
+    const initializeOpyPayment = useCallback((options: PaymentInitializeOptions) => initializePayment({
+        ...options,
+        opy: {
+            containerId: 'learnMoreButton',
+        },
+    }), [initializePayment]);
+
+    return (
+      <LoadingOverlay hideContentWhenLoading isLoading={ isInitializing }>
+        <strong>Buy now. Pay smarter.</strong>
```

Comment:
i think this should be added to the translated file

Reply:
Thanks, I think it's some kind of slogan that I need to ask the Openpay team if it should be translated.

Reply:
Agreed with @JoseAlvarezHdz, this should be in the translation file

Reply:
At the beginning of the project they told us that English is the only supported language, anyway, I asked them but they have not yet confirmed if their slogan needs to be translated.

Reply:
@animesh1987 there is some evidence they told us there is no need to translate, 
<img width="812" alt="image" src="https://user-images.githubusercontent.com/80552970/154180297-9e6b78f2-0e8d-421d-bd9c-d7f6007505fb.png">
you can find it here:
https://intranet.bigcommerce.com/pages/viewpage.action?pageId=145859374

I wonder if there would be any internal constraint for us to add translation anyways

Reply:
We have a centralised way of putting strings in the codebase, so you can just add it in en.json file if translation is not a concern. But we tend to avoid the above.

Reply:
Agree with Animesh. I also did this before. We could put text into `en.json` even we don't need translations. 
That's the way we manage text content.

Reply:
Ok, I didn't know it worked that way. Thanks, guys.